Remove  math/minuit2/.gitignore [skip-ci] (#5417)
root-project/root: math/minuit2/DEVELOP.md

This will fill in the `math/minuit2` directory with all the files needed for Minuit2, copied from the corresponding ROOT files, as part of the configure step.
At this point, you could continue to build (using `make`). Note that the CMake option `minuit2_inroot` will automatically be set to `ON` if you are inside the ROOT source tree. Setting `minuit2_standalone` requires that this be inside the ROOT source tree. As always, any manual setting of a cached variable in CMake will be remembered as long as the `CMakeCache.txt` file is not removed.

Remember that after building a tarball or a binary package you should remove the copied files using:

```bash
make purge
```

> Otehrwise the file will be visible as untracked files by git, unless you explicitly remove their tracking yourself with a .gitignore file

Reviewer: ```suggestion
Otherwise git shows the file as untracked, unless you explicitly remove their tracking yourself with a .gitignore file
```
